src/tinypeel/tinypeel.py

```diff
@@ -48,6 +56,10 @@ def runPeelingCycles(pedigree, peelingInfo, args, singleLocusMode=False):
                         peelingInfo.nLoci, 0.5, dtype=np.float32
                     )
     if args.est_alt_allele_prob:
+        if args.alt_allele_prob_file is not None:
+            warnings.warn(
+                "-est_alt_allele_prob will update the inputted alternative allele frequencies using all available observed genotypes. Therefore, will overwrite any differences between metafounders."
```

Comment:
@RosCraddock say that current implementation overwrites differences. We might improve upon this later …

Reply:
This is only true if a user uses both `—alt_allele_prob_file` and `—est_alt_allele_prob`. For now, I think I will rewrite this warning to recommend using `—update_alt_allele_prob` instead.